opencog/nlp/learn/run/run-server-mst.sh

```diff
@@ -0,0 +1,76 @@
+#! /bin/bash
+#
+# run-server-mst.sh
+#
+# Start cogserver on the local machine
+# This runs tmux with byobu to multiplex multiple terminals;
+# use F3 and F4 to swtich to the other terminals.
+#
+
+# Work around an lxc-attach bug.
+if [[ `tty` == "not a tty" ]]
+then
+	script -c $0 /dev/null
+	exit 0
+fi
+
+export LD_LIBRARY_PATH=/usr/local/lib/opencog/modules
+
+if [ $# -lt 2 ]
+then 
+  echo "Usage: ./run-server-mst.sh <language> <db_name> [<username>] [<password>]"
+  exit 0
+fi
+
+args=("$@")
+# Use byobu so that the scroll bars actually work
+byobu new-session -d -n 'cntl' '$SHELL'
+if [ $# -eq 2 ]
+then
+	byobu new-window -n 'cogsrv' "nice guile -l mst-launch.scm -- --lang $1 --db $2 ; $SHELL"
+elif [ $# -eq 3 ]
+then
+	byobu new-window -n 'cogsrv' "nice guile -l mst-launch.scm -- --lang $1 --db $2 --user $3; $SHELL"
+elif [ $# -eq 4 ]
+then
+	byobu new-window -n 'cogsrv' "nice guile -l mst-launch.scm -- --lang $1 --db $2 --user $3 --password $4; $SHELL"
+fi
+sleep 2;
+
+# Assign port value according to language
+if [ "$1" = "en" ]
+then
+  PORT=19005
+elif [ "$1" = "fr" ]
+then
+  PORT=19003
+elif [ "$1" = "lt" ]
+then
+  PORT=19002
+elif [ "$1" = "pl" ]
+then
+  PORT=19004
+elif [ "$1" = "yue" ]
+then
+  PORT=19006
+elif [ "$1" = "zh" ]
```

Comment:
All this is crazy overkill. There are thousands of languages; we won't be listing them all here. If and when Anton wants to run the pipeline on russian, it does not make sense to edit this file to add Russian.   Its very important in software to keep things as simple as possible. This is adding complexity for no reason. 

I'm also nervous about the lang-username-password flags in the scripts as well. They make the scripts long, verbose, difficult to understand, difficult to debug.  The system is already complex, we should not be adding additional complexity, when there is little or no point to it.

Reply:
Linas, I just added them for completeness for the languages that you already had pre-set. I agree that there is no point in having this, but you were the one who introduced it.
About the lang-username-passwords, it really makes it easier to introduce these in command line, rather than going into a few files and changing them manually each time I want to test a new corpus or a different database. The language part is, again, to comply with your multi-language requirements.